database_scripts/prod_data_backup.sql

```diff
@@ -0,0 +1,97 @@
+drop table manual_claim_clear_down;
+
+create table manual_claim_clear_down as
+(select c.id     "claim_id",
+ 		ecpe.id  "eligibility_id"
+  from public.early_career_payments_eligibilities ecpe,
+	   public.claims c
+ where ecpe.id = c.eligibility_id);
+
+ alter table manual_claim_clear_down
+   add primary key (claim_id, eligibility_id);
+
+insert into manual_claim_clear_down (claim_id, eligibility_id) 
+select c.id "claim_id",
+	   a.id "eligibility_id"
+  from public.student_loans_eligibilities a,
+  	   public.claims c
+ where a.id = c.eligibility_id
+   and a.created_at BETWEEN '2021-04-01T00:00:01' and '2021-10-29T14:33:14';
```

Comment:
@and-lucas-kelly any reason this is not 31-August-2021?

Reply:
updated to end of first week in September as that is the last possible date for Private beta